### PR TITLE
uploading prob 6and 7

### PR DESCRIPTION
**Pull request by humeratamkin**

### CODE REVIEW

File: Personel/humera/programs/sept7/prob6.c [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/* c program to add two numbers using pointers */
+
+#include <stdio.h>
+int main()
+
```
Comment (Harshada-github) [inline Personel/humera/programs/sept7/prob6.c:5]: remove extra line spaces

**Review by PrajwalYadav — COMMENTED** [review]
remove extra spaces 
remove comments from code other than name of program

**Review by PrajwalYadav — COMMENTED** [review]
coding style should arrange properly

File: Team_02/linkstd.c [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+#include<malloc.h>
+ struct node
+  {
+     char name[29];
+     int mark;
+     int roll;
+     struct node *next;
+  };
+
+  int main()
```
Comment (AdeshK1331) [inline Team_02/linkstd.c:12]: main

File: Team_02/linkstd.c [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+while(1)
+ {
+     
+      printf("\n1.display\n2:add\n3.delete\n4.modification\n5.exit\n");
+        printf("\nenter your choice=");
+      scanf("%d",&ch);
+      switch(ch)
+     
+{
```
Comment (AdeshK1331) [inline Team_02/linkstd.c:32]: check the coding style.

**Review by AdeshK1331 — COMMENTED** [review]
something

**Review by AdeshK1331 — CHANGES_REQUESTED** [review]
if

**Review by AdeshK1331 — APPROVED** [review]

**Pull request merged**